### PR TITLE
update Repository Configuration JSON

### PR DESCRIPTION
**Pull request by mfominov**

### CODE REVIEW

**Comment by pratz** [comment]
@mfominov thanks for you contribution, appreciate it 👍

**Comment by pratz** [comment]
@fuzzyhandle @ParagDoke @varungituser can anyone of you guys merge this PR, apparently my permissions are revoked.
I have tested this PR on my local machine and seems good 👍

File: artifactory/repository/local.py [hunk 1]
```diff
@@ -37,6 +38,9 @@ class Local(RepositoryType):
             ("yum_root_depth", "yumRootDepth", 0),
             ("docker_api_version", "dockerApiVersion", "V2"),
             ("force_docker_authentication", "forceDockerAuthentication", False),
+            ("enable_file_lists_indexing", "enableFileListsIndexing", False),
+            ("optional_index_compression_formats", "optionalIndexCompressionFormats", ["bz2"]),
+            ("downloa_dredirect", "downloadRedirect", False),
```
Comment (ParagDoke) [inline artifactory/repository/local.py]: Probably a typo

Comment (mfominov) [reply artifactory/repository/local.py]: thx a lot.

**Pull request merged**